Program.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+import java.util.regex.Pattern;
+public class Program {
+ 
+    public static void main(String[] args) {
+       Scanner in = new Scanner (System.in); 
+       System.out.print("Input a sentence: ");
+       String sentence = in.nextLine();
+       in.close();
+       StringBuffer strBuf = new StringBuffer (sentence);
+       
+       RevOrder (strBuf);
+       System.out.println("==========");
+       Words (sentence);
+       System.out.println("==========");
+       ReplaceSent (sentence);
+       System.out.println("==========");
+       GettingSent (sentence);
+       System.out.println("==========");
+       
+      
+    }
+    
+    //Reverse Order
+    static void RevOrder (StringBuffer Order) {
+    	Order.reverse();
+    	System.out.print("Reverse order: ");
+    	System.out.println(Order.toString());
+    }
+    
+    //Split by Words
+    static void Words (String sentence) {
+    	String[] words = sentence.split("\\s*(\\s|,|!|\\.)\\s*");
```

Comment:
have you find this regular expression for string breaks in google or create it by your own? I don't recommend to use something you don't know exactly how it works.